src/components/InputWithTooltip.tsx

```diff
@@ -68,9 +68,7 @@ const InputWithTooltip: React.RefForwardingComponent<HTMLInputElement, InputProp
   return (
     <>
       <Input $error={showErrorStyle} {...props} {...finalTargetProps} />
-      <Tooltip bgColor={tooltipBgColor} {...tooltipProps}>
```

Comment:
Instead of passing `bgColor` as a prop, now we base it on a `theme` variable (shared between dark/light mode).

Reply:
Opting for simplifying this tooltip component, until we decide we need additional props?

Reply:
This component is also legacy. I would rather move it to `apps/gp-v1/components/common` as well and use the legacy tooltip component to avoid any issues we might have if/when we decide to release a new version of the old interface.

Reply:
Ah I see. Good if I address this in this PR?

Reply:
Sure, I don't see a problem doing it here.